### ISSUE TITLE
注释掉的代码

### ISSUE THREAD

**Issue by lyriccs** [broliang/HRGAT#5]
请问这段代码为什么注释掉了，我把它加上之后一直报错
# if self.use_text:
        #     edge_data_t = self.comp(edges.src['t'], self.rel[edge_type])  # [E, in_channel]
        #     msg_t = torch.cat([torch.matmul(edge_data_t[:edge_num // 2, :], self.in_w_t),
        #                        torch.matmul(edge_data_t[edge_num // 2:, :], self.out_w_t)])
        #     msg_t = torch.cat((Variable(torch.ones(msg_t.shape[0], 1).type(self.DTYPE).to(self.device), requires_grad=False), msg_t), dim=1)
        #     fused_msg_t = torch.matmul(msg_t, self.text_facotr)
        #
        # if self.use_img:
        #     edge_data_i = self.comp(edges.src['i'], self.rel[edge_type])  # [E, in_channel]
        #     msg_i = torch.cat([torch.matmul(edge_data_i[:edge_num // 2, :], self.in_w_i),
        #                        torch.matmul(edge_data_i[edge_num // 2:, :], self.out_w_i)])
        #     msg_i = torch.cat((Variable(torch.ones(msg_i.shape[0], 1).type(self.DTYPE).to(self.device), requires_grad=False), msg_i), dim=1)
        #     fused_msg_i = torch.matmul(msg_i, self.img_facotr)
        #
        #     # msg = torch.cat([msg, msg_i], dim=1)

**Comment by ttzhang0417** [comment]
您好，请问你解决这个问题了吗，我也有同样的疑问

**Comment by lyriccs** [comment]
我正在改这段代码



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年5月3日(周五) 晚上7:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [broliang/HRGAT] 注释掉的代码 (Issue #5)




 
您好，请问你解决这个问题了吗，我也有同样的疑问
